### PR TITLE
Show category

### PR DESCRIPTION
**Pull request by kagoromo**

### CODE REVIEW

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -12,6 +12,8 @@ def show
     Settings.word.number_of_answers.times do
       @answer = @word.answers.build
     end
+    @words = @category.words.paginate(:page => params[:page],
```
Comment (vietmt) [inline app/controllers/admin/categories_controller.rb]: Remove ()

File: app/views/admin/words/_word.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    </tr>
+    
+    <% word.answers.each do |answer| %>
+      <tr>
+        <td>
+          <%= answer.answer %>
+        </td>
+        <td>
+          <% if answer.is_correct %>
+            Correct
```
Comment (vietmt) [inline app/views/admin/words/_word.html.erb]: I18n

File: app/controllers/admin/categories_controller.rb (hunk 1 above)
Comment (hellovietnam93) [inline app/controllers/admin/categories_controller.rb]: convention em ơi

File: app/views/admin/words/_word.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <tr>
+      <td class="col-md-10">Choices</td>
+      <td class="col-md-2">Correct</td>
+    </tr>
+    <% word.answers.each do |answer| %>
+      <tr>
+        <td><%= answer.answer %></td>
+        <td>
+          <% if answer.is_correct %>
+            Yes
```
Comment (hellovietnam93) [inline app/views/admin/words/_word.html.erb]: I18n

File: app/views/admin/words/_word.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+          <% end %>
+        </td>
+      </tr>  
+    <% end %>
```
Comment (hellovietnam93) [inline app/views/admin/words/_word.html.erb]: ident

**Pull request merged**